MobileCenter/MobileCenter/Internals/AppDelegate/MSAppDelegateForwarder.m

```diff
@@ -179,17 +198,40 @@ + (IMP)swizzleOriginalSelector:(SEL)originalSelector
      * depend on the SDK return value for its own logic so customers already have to call the SDK API
      * in their implementation which makes swizzling useless.
      */
-    if (![self.selectorsNotToOverride containsObject:originalSelectorString]) {
+    if (![self.selectorsNotToOverride containsObject:originalSelectorStr]) {
```

Comment:
It's just 3 more letters, why not write them? ;)

Reply:
I usually shorten local variables name when they start to get big, I felt it was the case here. There are couple of other examples in the this page.